### DISCUSSION TITLE
There should seriously be a button for "don't go easy on me"

### DISCUSSION THREAD

**Discussion by Erenussocrates** [Monika-After-Story/MonikaModDev#9530] [general]
It's just frustrating someone does knowingly hold back against you while playing a game, and I fucking hate that, if someone would do that to me, I just wouldn't feel like playing with them again. Seriously, consider the players' feelings and seriously consider making a dialogue option for telling Monika to do as best as she can. It's just frustrating and pointless to see Monika saying "will go easier on you next time" when you are only ever playing against her anyway.

**Comment by Justformas** [comment]
What game are you talking about? All of them? Pong would get ridiculously difficult, unless you just mean don't go easier if you lose, only harder if you win. But even then at some point you'll reach a hard level and may only rarely be able to win if at all, and you might not want to stay stuck in a situation like that. Otherwise a button asking to go harder (or easier for that matter) could make sense.

Chess difficulty is kind of bugged, as it can be hard or easier at times. But once it's fixed, highest level will probably be very hard for most people. But I think it's also planned to initially be able to tell Monika your skill level, so she can start at a suitable level for you. 

I think your idea could work with NOU, as long as it's optional. 

But just so you know, the whole point of it getting easier is to consider the player's feelings, like so they don't get frustrated and can have a good experience. Some people may not want it to work like that, that's fine, but others do. Some people even may not like the idea of it working like that (or knowing that it does), but may still have a worse experience if it worked differently.

**Comment by Erenussocrates** [comment]
Pong is fine I guess, or there could be a "supposed maximum limit" to her artificial skill level that is still beatable by a human to her playing, which I assume there already exists anyway. But both at pong and especially in chess, I see her going with kid gloves whenever she achieved victory beforehand, and I thought that was ridiculous. 

I think players' feelings aren't the same for everyone. I can't just learn by someone going easy on me anyway. There could optionally be a button to tell Monika to "don't hold back" or "go easy on me" depending on player's mood.

**Reply by Justformas** [reply]
In GitHub discussions (rather than "issues"), you should use the reply/write a reply option if you are replying or talking about the same subject instead of writing a new unattached comment. Yeah it's a GitHub thing but it makes sense.

I seriously think there might be no limit to pong difficulty or it's extremely high. But right now that doesn't really matter because no one will get that high unless they somehow keep winning. However you might be saying Pong is noticeably easier because if you lost your last game before you stop playing it, it will indeed be noticeably easier next time (just for the next game or two). But if you instead win that last game then it will be noticeably harder next time (just for the next game).

In Chess she can make bad moves at practically any time, regardless of difficulty. Or she might not make many bad moves, regardless of difficulty. It's part of the reason why it's kind of bugged, but bad moves do seem more common on the lower levels. It's known how to fix and improve it though; it just might take some time to implement in the mod.

In case you were wondering, while there is something that affects difficulty in NOU, I think winning and losing has more to do with luck (and how you play).

----------------------------------------------------------------------------------

But I agree that players' feelings aren't the same for everyone. Some may want to play or learn against high difficulty even if they keep losing, but others may not or get discouraged against it. An idea like this has been brought up before, but I like the idea of being able to ask Monika to go/keep going harder or easier, for any game that Monika can play at different skill levels. Might not want it to sound like selecting a difficulty level in a game, but that can be avoided by making it like a natural conversation.
